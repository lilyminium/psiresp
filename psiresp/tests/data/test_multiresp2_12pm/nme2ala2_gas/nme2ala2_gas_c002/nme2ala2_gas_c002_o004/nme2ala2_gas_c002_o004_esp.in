memory 500 MB
molecule nme2ala2_gas_c002_o004 {
    units Angstrom
    no_com
    no_reorient
    0 1
        C                4.543572651507875     3.951302140450825     0.028256069939445
    H                5.062003524759239     3.027766442918551    -0.198192218210915
    H                4.836635911499604     4.714503702142354    -0.680520768458020
    H                4.840507466514140     4.285764582453264     1.016369327001167
    C                3.034217526460234     3.819829527188979    -0.000481442215386
    O                2.329486519376167     4.797461147326991    -0.000000000000006
    N                2.569129124851986     2.555124780207242    -0.004986772482991
    H                3.212374101901529     1.795156182534163    -0.003293893492073
    C                1.168743822081987     2.162568396959999     0.001598041306457
    C                0.458407626840532     2.672511764161948    -1.264915951960255
    H                0.525979876023733     3.749601246863697    -1.304562306050479
    H                0.925949041956632     2.260448501113395    -2.152704772912610
    H               -0.590705069470624     2.396599565318293    -1.272757344002712
    C                0.469675474302996     2.668236153156015     1.276146917616146
    H               -0.578750886114609     2.390276270025149     1.293173219852801
    H                0.946133116073740     2.254468855927579     2.158445283872082
    H                0.535767708615939     3.745376872113639     1.318028619343941
    C                1.192307264705037     0.620092106778468     0.000087387543414
    O                2.226011570791368     0.000000000000001     0.000000000000017
    N                0.000000000000000     0.000000000000000     0.000000000000000
    H               -0.835247130730830     0.534268804925532    -0.001398678005198
    C               -0.096543532981913    -1.445501271310071    -0.000994362660850
    H                0.376342852087454    -1.863989537839435    -0.879920500732333
    H                0.379595024440315    -1.865079800916531     0.875604943783017
    H               -1.142898904445997    -1.718770663711228     0.000788753280134


}

set basis 6-31g*
E, wfn = prop('hf', properties=['GRID_ESP'], return_wfn=True)
esp = wfn.oeprop.Vvals()
