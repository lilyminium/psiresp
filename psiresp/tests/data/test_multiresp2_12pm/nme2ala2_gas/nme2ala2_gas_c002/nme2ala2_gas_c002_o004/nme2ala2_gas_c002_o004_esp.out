
    -----------------------------------------------------------------------
          Psi4: An Open-Source Ab Initio Electronic Structure Package
                               Psi4 1.3.2 release

                         Git: Rev {HEAD} ecbda83 


    R. M. Parrish, L. A. Burns, D. G. A. Smith, A. C. Simmonett,
    A. E. DePrince III, E. G. Hohenstein, U. Bozkaya, A. Yu. Sokolov,
    R. Di Remigio, R. M. Richard, J. F. Gonthier, A. M. James,
    H. R. McAlexander, A. Kumar, M. Saitow, X. Wang, B. P. Pritchard,
    P. Verma, H. F. Schaefer III, K. Patkowski, R. A. King, E. F. Valeev,
    F. A. Evangelista, J. M. Turney, T. D. Crawford, and C. D. Sherrill,
    J. Chem. Theory Comput. 13(7) pp 3185--3197 (2017).
    (doi: 10.1021/acs.jctc.7b00174)


                         Additional Contributions by
    P. Kraus, H. Kruse, M. H. Lechner, M. C. Schieber, R. A. Shaw,
    A. Alenaizan, R. Galvelis, Z. L. Glick, S. Lehtola, and J. P. Misiewicz

    -----------------------------------------------------------------------


    Psi4 started on: Sunday, 25 July 2021 01:20AM

    Process ID: 1593083
    Host:       gavle
    PSIDATADIR: /store/opt/anaconda3/envs/psiresp/share/psi4
    Memory:     500.0 MiB
    Threads:    1
    
  ==> Input File <==

--------------------------------------------------------------------------
memory 500 MB
molecule nme2ala2_gas_c002_o004 {
    units Angstrom
    no_com
    no_reorient
    0 1
        C                4.543572651507875     3.951302140450825     0.028256069939445
    H                5.062003524759239     3.027766442918551    -0.198192218210915
    H                4.836635911499604     4.714503702142354    -0.680520768458020
    H                4.840507466514140     4.285764582453264     1.016369327001167
    C                3.034217526460234     3.819829527188979    -0.000481442215386
    O                2.329486519376167     4.797461147326991    -0.000000000000006
    N                2.569129124851986     2.555124780207242    -0.004986772482991
    H                3.212374101901529     1.795156182534163    -0.003293893492073
    C                1.168743822081987     2.162568396959999     0.001598041306457
    C                0.458407626840532     2.672511764161948    -1.264915951960255
    H                0.525979876023733     3.749601246863697    -1.304562306050479
    H                0.925949041956632     2.260448501113395    -2.152704772912610
    H               -0.590705069470624     2.396599565318293    -1.272757344002712
    C                0.469675474302996     2.668236153156015     1.276146917616146
    H               -0.578750886114609     2.390276270025149     1.293173219852801
    H                0.946133116073740     2.254468855927579     2.158445283872082
    H                0.535767708615939     3.745376872113639     1.318028619343941
    C                1.192307264705037     0.620092106778468     0.000087387543414
    O                2.226011570791368     0.000000000000001     0.000000000000017
    N                0.000000000000000     0.000000000000000     0.000000000000000
    H               -0.835247130730830     0.534268804925532    -0.001398678005198
    C               -0.096543532981913    -1.445501271310071    -0.000994362660850
    H                0.376342852087454    -1.863989537839435    -0.879920500732333
    H                0.379595024440315    -1.865079800916531     0.875604943783017
    H               -1.142898904445997    -1.718770663711228     0.000788753280134


}

set basis 6-31g*
E, wfn = prop('hf', properties=['GRID_ESP'], return_wfn=True)
esp = wfn.oeprop.Vvals()
--------------------------------------------------------------------------

  Memory set to   2.794 GiB by Python driver.

  Memory set to 476.837 MiB by Python driver.

*** tstart() called on gavle
*** at Sun Jul 25 01:20:13 2021

   => Loading Basis Set <=

    Name: 6-31G*
    Role: ORBITAL
    Keyword: BASIS
    atoms 1, 5, 9-10, 14, 18, 22          entry C          line   111 file /store/opt/anaconda3/envs/psiresp/share/psi4/basis/6-31gs.gbs 
    atoms 2-4, 8, 11-13, 15-17, 21, 23-25 entry H          line    44 file /store/opt/anaconda3/envs/psiresp/share/psi4/basis/6-31gs.gbs 
    atoms 6, 19                           entry O          line   145 file /store/opt/anaconda3/envs/psiresp/share/psi4/basis/6-31gs.gbs 
    atoms 7, 20                           entry N          line   128 file /store/opt/anaconda3/envs/psiresp/share/psi4/basis/6-31gs.gbs 


         ---------------------------------------------------------
                                   SCF
               by Justin Turney, Rob Parrish, Andy Simmonett
                          and Daniel G. A. Smith
                              RHF Reference
                        1 Threads,    476 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c1
    Full point group: C1

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z               Mass       
    ------------   -----------------  -----------------  -----------------  -----------------
         C            4.543572651508     3.951302140451     0.028256069939    12.000000000000
         H            5.062003524759     3.027766442919    -0.198192218211     1.007825032230
         H            4.836635911500     4.714503702142    -0.680520768458     1.007825032230
         H            4.840507466514     4.285764582453     1.016369327001     1.007825032230
         C            3.034217526460     3.819829527189    -0.000481442215    12.000000000000
         O            2.329486519376     4.797461147327    -0.000000000000    15.994914619570
         N            2.569129124852     2.555124780207    -0.004986772483    14.003074004430
         H            3.212374101902     1.795156182534    -0.003293893492     1.007825032230
         C            1.168743822082     2.162568396960     0.001598041306    12.000000000000
         C            0.458407626841     2.672511764162    -1.264915951960    12.000000000000
         H            0.525979876024     3.749601246864    -1.304562306050     1.007825032230
         H            0.925949041957     2.260448501113    -2.152704772913     1.007825032230
         H           -0.590705069471     2.396599565318    -1.272757344003     1.007825032230
         C            0.469675474303     2.668236153156     1.276146917616    12.000000000000
         H           -0.578750886115     2.390276270025     1.293173219853     1.007825032230
         H            0.946133116074     2.254468855928     2.158445283872     1.007825032230
         H            0.535767708616     3.745376872114     1.318028619344     1.007825032230
         C            1.192307264705     0.620092106778     0.000087387543    12.000000000000
         O            2.226011570791     0.000000000000     0.000000000000    15.994914619570
         N            0.000000000000     0.000000000000     0.000000000000    14.003074004430
         H           -0.835247130731     0.534268804926    -0.001398678005     1.007825032230
         C           -0.096543532982    -1.445501271310    -0.000994362661    12.000000000000
         H            0.376342852087    -1.863989537839    -0.879920500732     1.007825032230
         H            0.379595024440    -1.865079800917     0.875604943783     1.007825032230
         H           -1.142898904446    -1.718770663711     0.000788753280     1.007825032230

  Running in c1 symmetry.

  Rotational constants: A =      0.07400  B =      0.00924  C =      0.00870 [cm^-1]
  Rotational constants: A =   2218.55680  B =    276.90300  C =    260.91897 [MHz]
  Nuclear repulsion =  667.373983404715318

  Charge       = 0
  Multiplicity = 1
  Electrons    = 86
  Nalpha       = 43
  Nbeta        = 43

  ==> Algorithm <==

  SCF Algorithm Type is DF.
  DIIS enabled.
  MOM disabled.
  Fractional occupation disabled.
  Guess Type is SAD.
  Energy threshold   = 1.00e-06
  Density threshold  = 1.00e-06
  Integral threshold = 0.00e+00

  ==> Primary Basis <==

  Basis Set: 6-31G*
    Blend: 6-31G*
    Number of shells: 94
    Number of basis function: 193
    Number of Cartesian functions: 193
    Spherical Harmonics?: false
    Max angular momentum: 2

   => Loading Basis Set <=

    Name: (6-31G* AUX)
    Role: JKFIT
    Keyword: DF_BASIS_SCF
    atoms 1, 5, 9-10, 14, 18, 22          entry C          line   121 file /store/opt/anaconda3/envs/psiresp/share/psi4/basis/cc-pvdz-jkfit.gbs 
    atoms 2-4, 8, 11-13, 15-17, 21, 23-25 entry H          line    51 file /store/opt/anaconda3/envs/psiresp/share/psi4/basis/cc-pvdz-jkfit.gbs 
    atoms 6, 19                           entry O          line   221 file /store/opt/anaconda3/envs/psiresp/share/psi4/basis/cc-pvdz-jkfit.gbs 
    atoms 7, 20                           entry N          line   171 file /store/opt/anaconda3/envs/psiresp/share/psi4/basis/cc-pvdz-jkfit.gbs 

  ==> Pre-Iterations <==

   -------------------------------------------------------
    Irrep   Nso     Nmo     Nalpha   Nbeta   Ndocc  Nsocc
   -------------------------------------------------------
     A        193     193       0       0       0       0
   -------------------------------------------------------
    Total     193     193      43      43      43       0
   -------------------------------------------------------

  ==> Integral Setup <==

  DFHelper Memory: AOs need 0.332 GiB; user supplied 0.349 GiB. Using in-core AOs.

  ==> MemDFJK: Density-Fitted J/K Matrices <==

    J tasked:                   Yes
    K tasked:                   Yes
    wK tasked:                   No
    OpenMP threads:               1
    Memory [MiB]:               357
    Algorithm:                 Core
    Schwarz Cutoff:           1E-12
    Mask sparsity (%):       9.3049
    Fitting Condition:        1E-10

   => Auxiliary Basis Set <=

  Basis Set: (6-31G* AUX)
    Blend: CC-PVDZ-JKFIT
    Number of shells: 390
    Number of basis function: 1241
    Number of Cartesian functions: 1241
    Spherical Harmonics?: false
    Max angular momentum: 3

  Minimum eigenvalue in the overlap matrix is 1.5127616091E-03.
  Using Symmetric Orthogonalization.

  SCF Guess: Superposition of Atomic Densities via on-the-fly atomic UHF.

  ==> Iterations <==

                           Total Energy        Delta E     RMS |[F,P]|

   @DF-RHF iter SAD:  -530.67052853614746   -5.30671e+02   0.00000e+00 
   @DF-RHF iter   1:  -531.58764937003343   -9.17121e-01   3.98550e-03 DIIS
   @DF-RHF iter   2:  -531.81140728960054   -2.23758e-01   2.38173e-03 DIIS
   @DF-RHF iter   3:  -531.88627587888993   -7.48686e-02   4.58501e-04 DIIS
   @DF-RHF iter   4:  -531.89103910476229   -4.76323e-03   1.65916e-04 DIIS
   @DF-RHF iter   5:  -531.89170737375309   -6.68269e-04   3.39814e-05 DIIS
   @DF-RHF iter   6:  -531.89176822536172   -6.08516e-05   1.76962e-05 DIIS
   @DF-RHF iter   7:  -531.89178121045950   -1.29851e-05   6.30646e-06 DIIS
   @DF-RHF iter   8:  -531.89178348145037   -2.27099e-06   1.86995e-06 DIIS
   @DF-RHF iter   9:  -531.89178365838643   -1.76936e-07   6.32815e-07 DIIS
  Energy and wave function converged.


  ==> Post-Iterations <==

    Orbital Energies [Eh]
    ---------------------

    Doubly Occupied:                                                      

       1A    -20.536807     2A    -20.512203     3A    -15.595830  
       4A    -15.574103     5A    -11.354915     6A    -11.342142  
       7A    -11.292144     8A    -11.265964     9A    -11.224950  
      10A    -11.221310    11A    -11.221187    12A     -1.407008  
      13A     -1.380288    14A     -1.244125    15A     -1.234478  
      16A     -1.074789    17A     -0.989642    18A     -0.962998  
      19A     -0.950373    20A     -0.871521    21A     -0.808569  
      22A     -0.754521    23A     -0.735708    24A     -0.663294  
      25A     -0.656512    26A     -0.642794    27A     -0.627404  
      28A     -0.621773    29A     -0.599891    30A     -0.589162  
      31A     -0.586045    32A     -0.565873    33A     -0.561994  
      34A     -0.559098    35A     -0.546355    36A     -0.530554  
      37A     -0.514523    38A     -0.512229    39A     -0.486931  
      40A     -0.425459    41A     -0.405472    42A     -0.402741  
      43A     -0.380400  

    Virtual:                                                              

      44A      0.192439    45A      0.199285    46A      0.213991  
      47A      0.248202    48A      0.256535    49A      0.269541  
      50A      0.293220    51A      0.304585    52A      0.310572  
      53A      0.312816    54A      0.328249    55A      0.336830  
      56A      0.338858    57A      0.343095    58A      0.358357  
      59A      0.363144    60A      0.375481    61A      0.388783  
      62A      0.406101    63A      0.430037    64A      0.446839  
      65A      0.475668    66A      0.520736    67A      0.550736  
      68A      0.624384    69A      0.638650    70A      0.694002  
      71A      0.721697    72A      0.726525    73A      0.735541  
      74A      0.742681    75A      0.760448    76A      0.766055  
      77A      0.775703    78A      0.800944    79A      0.834083  
      80A      0.860054    81A      0.878450    82A      0.906534  
      83A      0.916211    84A      0.934152    85A      0.938851  
      86A      0.985807    87A      0.999774    88A      1.009101  
      89A      1.028445    90A      1.033824    91A      1.037418  
      92A      1.049513    93A      1.107811    94A      1.120209  
      95A      1.122096    96A      1.125917    97A      1.139782  
      98A      1.148818    99A      1.169182   100A      1.181053  
     101A      1.185429   102A      1.194034   103A      1.202963  
     104A      1.212079   105A      1.218117   106A      1.223386  
     107A      1.230682   108A      1.241853   109A      1.257225  
     110A      1.277786   111A      1.281629   112A      1.307625  
     113A      1.360911   114A      1.373368   115A      1.420700  
     116A      1.456819   117A      1.492135   118A      1.532472  
     119A      1.553743   120A      1.565685   121A      1.602863  
     122A      1.629965   123A      1.649512   124A      1.671458  
     125A      1.748748   126A      1.754595   127A      1.759914  
     128A      1.788493   129A      1.805704   130A      1.922820  
     131A      1.936980   132A      1.991328   133A      2.040990  
     134A      2.051104   135A      2.061841   136A      2.108189  
     137A      2.130912   138A      2.144128   139A      2.149930  
     140A      2.167600   141A      2.173612   142A      2.203113  
     143A      2.215690   144A      2.253048   145A      2.255422  
     146A      2.276840   147A      2.321010   148A      2.335835  
     149A      2.364817   150A      2.381556   151A      2.408327  
     152A      2.421536   153A      2.448208   154A      2.491377  
     155A      2.518266   156A      2.559677   157A      2.580267  
     158A      2.590006   159A      2.610890   160A      2.611460  
     161A      2.632534   162A      2.701661   163A      2.710755  
     164A      2.726209   165A      2.760557   166A      2.780747  
     167A      2.810169   168A      2.861022   169A      2.871312  
     170A      2.952927   171A      2.973688   172A      3.009855  
     173A      3.036278   174A      3.087560   175A      3.145693  
     176A      3.168625   177A      3.283885   178A      3.342813  
     179A      3.389953   180A      3.410050   181A      3.545770  
     182A      3.633102   183A      4.399931   184A      4.451552  
     185A      4.520667   186A      4.536345   187A      4.649469  
     188A      4.705430   189A      4.724721   190A      4.729644  
     191A      4.937300   192A      4.962693   193A      5.194587  

    Final Occupation by Irrep:
              A 
    DOCC [    43 ]

  @DF-RHF Final Energy:  -531.89178365838643

   => Energetics <=

    Nuclear Repulsion Energy =            667.3739834047153181
    One-Electron Energy =               -2049.6630553330842304
    Two-Electron Energy =                 850.3972882699824822
    Total Energy =                       -531.8917836583864300

Computation Completed


Properties will be evaluated at   0.000000,   0.000000,   0.000000 [a0]

Properties computed using the HF density matrix

  Nuclear Dipole Moment: [e a0]
     X:   259.9202      Y:   318.7329      Z:     0.7036

  Electronic Dipole Moment: [e a0]
     X:  -260.7722      Y:  -319.6968      Z:    -0.6829

  Dipole Moment: [e a0]
     X:    -0.8520      Y:    -0.9639      Z:     0.0207     Total:     1.2866

  Dipole Moment: [D]
     X:    -2.1655      Y:    -2.4499      Z:     0.0527     Total:     3.2702


 Electrostatic potential computed on the grid and written to grid_esp.dat

*** tstop() called on gavle at Sun Jul 25 01:20:25 2021
Module time:
	user time   =      12.31 seconds =       0.21 minutes
	system time =       0.12 seconds =       0.00 minutes
	total time  =         12 seconds =       0.20 minutes
Total time:
	user time   =      12.31 seconds =       0.21 minutes
	system time =       0.12 seconds =       0.00 minutes
	total time  =         12 seconds =       0.20 minutes

    Psi4 stopped on: Sunday, 25 July 2021 01:20AM
    Psi4 wall time for execution: 0:00:12.48

*** Psi4 exiting successfully. Buy a developer a beer!
